docs/gambit/mutation-verifier.md

```diff
@@ -18,19 +18,21 @@ Installation instructions can be found [here](https://www.java.com/en/download/h
 
 ## Running the Mutation Verifier
 
-- Example:
+You will need to update your `certora-cli-alpha-master` installation using `pip` to get the relevant
+dependencies. Then you can run Gambit like so:
+
 ```
-java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/example.conf
+mutationTest path/to/config/file/Example.conf
 ```
 
 - **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
```

Comment:
I would probably leave this out; it's not special for the mutation tester, every command line program has this problem.  Is there any reason to expect that this will come up?  I certainly wouldn't mention Kotlin here.